package.json

```diff
@@ -37,10 +38,12 @@
   "engines": {
     "node": ">=14.0.0"
   },
+  "types": "dist/src/index.d.ts",
   "devDependencies": {
-    "aegir": "^33.0.0",
+    "@types/debug": "^4.1.5",
+    "aegir": "^30.3.0",
```

Comment:
Why this downgrade?

Reply:
I found latest version of `aegir` does not support `Node.js` only builds. But I will check once again if I could find a way.

Reply:
We need to use latest aegir to have proper types being generated and to guarantee future fixes.

I think it should support, but if it does not we need to create an issue and get the support to it.

Reply:
Solved the issue with `aegir` update. FYI https://github.com/ipfs/aegir/blob/master/md/migration-to-v31.md#--node-flag-removed